channels/us.m3u

```diff
@@ -319,7 +289,7 @@ https://adultswim-vodlive.cdn.turner.com/live/aqua-teen/stream.m3u8
 https://arcade-cloud-br.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="ArcadeCloudMexico.us" tvg-name="Arcade Cloud (Mexico)" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Arcade Cloud (Mexico)
 https://arcade-cloud-mx.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="ArchitecturalDigest.us" tvg-name="Architectural Digest" tvg-country="US" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/AD_246x246.png?raw=true" group-title="",Architectural Digest
+#EXTINF:-1 tvg-id="Traveler.us" tvg-name="Traveler" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Traveler
```

Comment:
It is indeed ArchitecturalDigest, I've seen AD on the screen on a grid background. Also the content is all about design and architecture

Reply:
Interesting... they named it badly. Renaming it.